1.24/Dockerfile.c8s

```diff
@@ -85,9 +85,4 @@ USER 1001
 
 STOPSIGNAL SIGQUIT
 
-# Not using VOLUME statement since it's not working in OpenShift Online:
```

Comment:
If I understand this correctly, the issue with volumes is no longer present on OCP.
In that case, don't we want to use the VOLUMEs in Containerfiles instead of removing them entirely?

Reply:
@bparees Can you please respond more clearly to why we do not need to define VOLUME directive, I guess I explained it but you will be more precise. Thank you

Reply:
They were really never needed.  VOLUME defines implicit volume mounts, but it's better to just use explicit ones (meaning either using "-v" when running an image via docker, or using a volume/volumemount in the pod definition in k8s/openshift).  So you could always mount a filesystem to a path in a container, even if the image didn't explicitly define a VOLUME. 

the problem was that when docker saw an image with a VOLUME definition it would automatically try to mount a volume(empty) into that path, which openshift online did not allow.  But i do not believe that behavior is part of OCP any more with the move off of docker as the container runtime.

It is somewhat similar to the "EXPOSE" declaration....it's more of a hint than a necessary declaration.  The difference is EXPOSE does literally nothing at runtime, whereas VOLUME does cause a volume to be mounted at least with docker's container runtime.

some more background: https://stackoverflow.com/questions/40163036/difference-between-volume-declaration-in-dockerfile-and-v-as-docker-run-paramet

Reply:
the other thing that changed is how openshift runs s2i builds. It no longer talks to the docker socket to launch a container running the s2i builder image.  So it's no longer relevant what VOLUMES the s2i builder image declares, so you won't hit this issue any more even if the image does declare a VOLUME.